scipy/optimize/_optimize.py

```diff
@@ -2354,15 +2354,14 @@ def optimize(self):
         #BEGIN CORE ALGORITHM
         #################################
         x = w = v = xb
-        fw = fv = fx = func(*((x,) + self.args))
+        fw = fv = fx = fb
```

Comment:
@mdhaber, I'm happy with the proposed change. `get_bracket_info` is returning `fa`, `fb`, `fc` that correspond to `xa`, `xb`, `xc`. So I don't think this function evaluation needs to happen again. If `bracket` isn't returning `f_i` corresponding to `x_i` then I think we have bigger problems.

Reply:
Yes, I suppose you're right.